### PR TITLE
Analog in

### PR DESCRIPTION
**Pull request by RikardO-HM**

### CODE REVIEW

File: docs/interfaces/c61/io.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+  - ADC
+  - Digital IO
+  - C61
+---
+
+## Overview
+
+This product has two analog inputs that is combined with two digital outputs available and two digital inputs on the same two connector pins.
+This analog in can be used to detect short circuit or to just monitor some signal between the supported voltage range of the platform.
```
Comment (albsod) [inline docs/interfaces/c61/io.md:12]:
```suggestion
This platform has two analog inputs that are combined with two digital outputs on the same connector pins.
Analog In can be used to detect short-circuits or to monitor certain signals within the supported voltage range of the platform.
```

File: docs/interfaces/mx4/analog.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    - [Start signals](start_signal.md)
+- T20, T30 and T30 FR
+    - input_voltage
+    - input_battery
+    - analog_1
+    - [Start signals](start_signal.md)
+
+
+```bash
+root@mx4-gtt:~# ls /opt/hm/pic_attributes/ | grep -i analog
```
Comment (albsod) [inline docs/interfaces/mx4/analog.md:40]:
```suggestion
# ls /opt/hm/pic_attributes/ | grep -i analog
```

File: docs/interfaces/mx4/analog.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Calibration files are not be used by end users.
+
+#### Example reading input voltage
+
+```bash
+root@mx4-gtt:~# cat /opt/hm/pic_attributes/input_voltage
+14917
+```
+
+#### Example when Vref (adc reference voltage) is turned off.
```
Comment (albsod) [inline docs/interfaces/mx4/analog.md:71]:
```suggestion
#### Example of when Vref (ADC reference voltage) is turned off.
```

File: docs/interfaces/mx4/analog.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+Vref should normally always be on but it is turned off when entering sleep
+([go_to_sleep.sh](https://github.com/hostmobility/mx4/blob/master/scripts/mx4/go_to_sleep.sh)).
+So should you poll adc values while entering you will get some errors after Vref
+has been turned off.
```
Comment (albsod) [inline docs/interfaces/mx4/analog.md:76]:
```suggestion
Vref should normally be always on but it is turned off when entering sleep
([go_to_sleep.sh](https://github.com/hostmobility/mx4/blob/master/scripts/mx4/go_to_sleep.sh)).
If you poll ADC values while entering sleep you will get some errors after Vref
has been turned off.
```

File: docs/interfaces/mx4/analog.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+```bash
+root@mx4-gtt:~# echo 0 > /sys/class/gpio/gpio243/value
+root@mx4-gtt:~# cat /opt/hm/pic_attributes/input_voltage
```
Comment (albsod) [inline docs/interfaces/mx4/analog.md:80]:
```suggestion
# echo 0 > /sys/class/gpio/gpio243/value
# cat /opt/hm/pic_attributes/input_voltage
```

File: docs/interfaces/mxv/analog.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  - MX-V
+---
+
+## Overview
+
+This platform uses IIO for read adc. The raw value is in mili volt and the list of supported input voltage.
+
+## List of supported Analog inputs
+
+DIG is also [digital input](digital_io.md) and KL15 is [start signals](start_signal.md) and KL30 is input power.
```
Comment (albsod) [inline docs/interfaces/mxv/analog.md:14]:
```suggestion
DIG is also [digital input](digital_io.md), KL15 is [start signals](start_signal.md), and KL30 is input power.
```

**Review by albsod — COMMENTED** [review]
I made the prompt shorter so as to remove implicit references to customers

File: docs/interfaces/mx4/analog.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+input_voltage_threshold_high
+input_voltage_threshold_low
+```
+
+Calibration files are not be used by end users.
+
+#### Example reading input voltage
+
+```bash
+root@mx4-gtt:~# cat /opt/hm/pic_attributes/input_voltage
```
Comment (RikardO-HM) [inline docs/interfaces/mx4/analog.md:67]:
```suggestion
cat /opt/hm/pic_attributes/input_voltage
```

**Pull request merged**